backend/src/sms/models/sms-message-transactional.ts

```diff
@@ -38,4 +39,13 @@ export class SmsMessageTransactional extends Model<SmsMessageTransactional> {
 
   @Column({ type: DataType.STRING, allowNull: true })
   messageId: string | null
+
+  @Column({
+    type: DataType.ENUM(...Object.values(MessageStatus)),
```

Comment:
It was intentional that we want to introduce a better message status system (than the existing `MessageStatus`) for transactional usage with an eventual goal of moving campaign message status to that as well

Reply:
Could you follow the API specs for the above reason?

Reply:
have refactored